app/app/services/aggregators/format_methods/argyle.rb

```diff
@@ -21,12 +21,21 @@ def self.format_date(date)
   end
 
   def self.format_currency(amount)
-    return unless amount
-    dollars, cents = amount.split(".").map(&:to_i)
+    return if amount.blank?
+    return unless amount.include?(".")
 
+    dollars, cents = amount.split(".").map(&:to_i)
     (dollars * 100) + cents
   end
 
+  def self.hours_computed(response_hours, response_gross_pay_list)
+    if response_hours.present? && response_hours.to_f > 0
+      response_hours
+    else
+      hours_by_earning_category(response_gross_pay_list)&.max_by { |_, value| value }&.[](1)
```

Comment:
`&.[](1)` could be more readable... Also, I think the Safe Navigation Operators should be unnecessary as the `hours_by_earning_category` should always return a hash. How about instead:

```suggestion
      hours_by_earning_category(response_gross_pay_list).map { |_category, hours| hours }.max
```